backend/zeno_backend/database/select.py

```diff
@@ -884,6 +887,15 @@ def project_state(project_uuid: str, project: Project) -> ProjectState | None:
         )
         models = [m[0] for m in model_results]
 
+        editor = db.execute_return(
+            "SELECT editor FROM user_project WHERE user_id = %s AND project_uuid = %s;",
+            [user.id, project_uuid],
+        )
+        if len(editor) == 0:
+            project.editor = False
+        else:
+            project.editor = bool(editor[0][0])
```

Comment:
Also need to check org_project I think. Similar to how we do it for the project/report.

Reply:
good catch